driver-core/src/main/com/mongodb/internal/logging/LogMessage.java

```diff
@@ -73,6 +99,10 @@ public enum Name {
             COMMAND_NAME("commandName"),
             REQUEST_ID("requestId"),
             OPERATION_ID("operationId"),
+            /**
+             * Not supported.
+             */
+            OPERATION("operation"),
```

Comment:
The final decision on whether to support `operation` and to what extent will be done in https://jira.mongodb.org/browse/JAVA-5197.